apps/web/src/components/showcase/showcase.css

```
@@ -0,0 +1,819 @@
+.showcase {
+  --showcase-accent: #a5b4fc;
+  color: #edf0f6;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)